<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lrc</class>
 <widget class="QDialog" name="lrc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1275</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListView" name="lw_lrcLrcList">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>951</width>
     <height>611</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lw_lrcMusicList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>831</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>610</y>
     <width>951</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QSlider" name="slider_lrcVolume">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>60</y>
      <width>20</width>
      <height>91</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_lrcPlayerAndPause">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="src.qrc">
      <normaloff>:/images/bfzn_play.png</normaloff>:/images/bfzn_play.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_lrcMusicTime">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="slider_lrcMusic">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_lrcVolume">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>120</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="src.qrc">
      <normaloff>:/images/voice_open.png</normaloff>:/images/voice_open.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_lrcNextMusic">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="src.qrc">
      <normaloff>:/images/bfzn_next.png</normaloff>:/images/bfzn_next.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_lrcLastMusic">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="src.qrc">
      <normaloff>:/images/bfzn_prev.png</normaloff>:/images/bfzn_prev.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_lrcPlayerStytle">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="src.qrc">
      <normaloff>:/images/orderPlayer.png</normaloff>:/images/orderPlayer.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_hideLrc">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>0</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>V</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
